tests/it/parser.rs

```diff
@@ -0,0 +1,225 @@
+// -*- coding: utf-8 -*-
+// ------------------------------------------------------------------------------------------------
+// Copyright © 2021, tree-sitter authors.
+// Licensed under either of Apache License, Version 2.0, or MIT license, at your option.
+// Please see the LICENSE-APACHE or LICENSE-MIT files in this distribution for license details.
+// ------------------------------------------------------------------------------------------------
+
+use tree_sitter_graph::ast::*;
+use tree_sitter_graph::Context;
+
+#[test]
+fn can_parse_blocks() {
```

Comment:
Would love to see smaller, expression- or statement-focused tests if possible, but otherwise this seems reasonable (and I am glad to see any tests at all 👍🏻).

Reply:
Yeah the wrinkle is that as written, this test case can only use the crate's public API, and I purposefully didn't expose parsing functions for all of the internal non-terminals in the DSL parser.  I could maybe write helper functions in here that wrap up an expression in a minimal "surrounding file" or something like that, but it wasn't obviously worth the effort compared to just making sure there's instances of all interesting DSL patterns that we might want to parse.